### ISSUE TITLE
Can you fix the Chinese lag, the game is very stuck in the Chinese language

### ISSUE THREAD

**Issue by liveskai** [momo5502/boiii#57]
**Can you fix the Chinese lag, the game is very stuck in the Chinese language**

**Comment by momo5502** [comment]
What do you mean by "very stuck"?

**Comment by liveskai** [comment]
> What do you mean by "very stuck"?

Frequent stops

**Comment by liveskai** [comment]
> What do you mean by "very stuck"?

This is the Chinese language file, play the game in Chinese, you will know what the problem is
https://musetransfer.com/s/07w5jj8ei

**Comment by momo5502** [comment]
I'm not sure if i want to support any version besides english, yet. Will have to see.

**Comment by esp-ra-de-alice-clones** [comment]
yes  Tradition and simple chinese subtitles will cause the game lag and stutter everytime the sub appears, no update and fix for this old game bug issue is real a shamed move for activition.....

**Comment by diamante0018** [comment]
@liveskai have you had a chance to test his again on latest boiii?

**Comment by liveskai** [comment]
> @liveskai have you had a chance to test his again on latest boiii?

yes

**Comment by diamante0018** [comment]
Still lags?

**Comment by liveskai** [comment]
> Still lags?

yes

**Comment by k7Ysh5A41** [comment]
> Still lags?

The game seems to be loading each character very slowly. During the loading process, the game is very stuck. If the loading is completed, the next time these characters appear again, there will be no freeze. This problem first appeared in 89 Version BO3, has been stuck since then, and this problem will appear in all non-latin character game languages

**Comment by k7Ysh5A41** [comment]
> Still lags?

There are some rumors that this issue was fixed by Aspyr in the macOS build of BO3, not sure if it is true or not

**Comment by Spidermanagent** [comment]
The game stutters in english as well, but not as serious as in other languages. There are frame spikes if you look at the frame generation graph, not sure what causes it but definitely not the hard drive (I put my game in nvme ssd). My initial suspecision would be shader generation and/or shader stutter. Secondly, it could be caused by CPU scheduling. I find a few settings in player config.ini that sets CPU vs frame generation that could attribute to resolution

**Comment by k7Ysh5A41** [comment]
> The game stutters in english as well, but not as serious as in other languages. There are frame spikes if you look at the frame generation graph, not sure what causes it but definitely not the hard drive (I put my game in nvme ssd). My initial suspecision would be shader generation and/or shader stutter. Secondly, it could be caused by CPU scheduling. I find a few settings in player config.ini that sets CPU vs frame generation that could attribute to resolution

The 88 version of BO3 runs extremely smoothly, it feels as smooth as the ps4 version, and this is the last time I can feel the smoothest BO3.

**Comment by Spidermanagent** [comment]
> > The game stutters in english as well, but not as serious as in other languages. There are frame spikes if you look at the frame generation graph, not sure what causes it but definitely not the hard drive (I put my game in nvme ssd). My initial suspecision would be shader generation and/or shader stutter. Secondly, it could be caused by CPU scheduling. I find a few settings in player config.ini that sets CPU vs frame generation that could attribute to resolution
> 
> The 88 version of BO3 runs extremely smoothly, it feels as smooth as the ps4 version, and this is the last time I can feel the smoothest BO3.

Sadly v88 isn't supported now, and it's missing patches, black market weapons and mod support. The only way is to fix it with t7 client, hopefully
